stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/response/AppSettingsResponse.kt

```diff
@@ -3,7 +3,9 @@ package io.getstream.chat.android.client.api2.model.response
 import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
-internal data class AppSettingsAPIResponse(val appDto: AppDto)
+internal data class AppSettingsResponse(
+    val app: AppDto,
```

Comment:
This was not working at all as the field name returned from the server is "appDto". So, I think breaking changes in the related logic are fine.